### DISCUSSION TITLE
Latest release v1.37.0 don't have windows distributive

### DISCUSSION THREAD

**Discussion by wovas** [jaegertracing/jaeger#3908] [Q&A] [answered]
https://github.com/jaegertracing/jaeger/releases/tag/v1.37.0 there no win package and link on [downloads page](https://www.jaegertracing.io/download/) doesn't work as well.

**Comment by albertteoh** [comment, marked as answer]
A similar issue was raised in https://github.com/jaegertracing/jaeger/issues/3871 and the decision was to wait for the next release (v1.38.0), which should come out this week.

**Reply by wovas** [reply]
Thanks @albertteoh.
In the meantime, previous release worked for me!
